src/pybind/mgr/dashboard/frontend/e2e/cluster/silences.po.ts

```diff
@@ -0,0 +1,122 @@
+    create: '/#/silence/create'
+  };
+
+  create(creator, comment, matcher_name, matcher_value) {
+    this.navigateTo('create');
+
+    expect(this.getBreadcrumbText()).toEqual('Create');
+
+    // enter silence creator
+    element(by.id('created-by')).clear();
+    element(by.id('created-by')).sendKeys(creator);
+
+    // enter silence comment
+    element(by.id('comment')).clear();
+    element(by.id('comment')).sendKeys(comment);
+
+    const addMatcherButton = element(by.cssContainingText('button', 'Add matcher'));
+    addMatcherButton.click().then(() => {
+      // select matcher name
+      const nameField = element(by.id('name'));
+      nameField.click();
+      expect(nameField.all(by.css('option')).getText()).toMatch(
+        matcher_name,
+        'Invalid matcher name'
+      );
+      nameField.element(by.cssContainingText('option', matcher_name)).click();
+
+      // enter matcher value
+      const valueField = element(by.id('value'));
+      valueField.click();
+      valueField.sendKeys(matcher_value);
+
+      const addButton = element.all(by.cssContainingText('button', 'Add')).last();
+      addButton.click().then(() => {
+        const createButton = element(by.cssContainingText('button', 'Create Silence'));
+        // wait for modal container where you select Matcher to dissapear
+        browser.wait(
+          Helper.EC.not(
+            Helper.EC.presenceOf(element(by.cssContainingText('.modal-dialog', 'Matcher')))
+          ),
+          5000
+        );
+        createButton.click().then(() => {
+          // wait for browser to return to silences page and table to be visible
+          browser.wait(
+            Helper.EC.presenceOf(this.getTable().first()),
+            Helper.TIMEOUT,
+            'Timed out waiting for silence creation'
+          );
+
+          // Order silences so most recently updated one is on top
+          element(by.cssContainingText('.datatable-header-cell-label.draggable', 'Updated'))
+            .click()
+            .then(() => {
+              element(
+                by.cssContainingText('.datatable-header-cell-label.draggable', 'Updated')
+              ).click();
+            });
+
+          // Opens new silence's details table by clicking on silence then check "comment"
+          // row of details table for silence for the correct comment to verify its existence
+          this.getFirstTableCellWithText(creator).click();
+          expect(element(by.cssContainingText('.datatable-body-row', 'comment')).getText()).toMatch(
+            comment
+          );
+        });
+      });
+    });
+  }
+
+  // expire is currently limited to expiring the most recently updated silence with the given creator name
+  expire(creator) {
+    this.navigateTo();
+
+    expect(this.getBreadcrumbText()).toEqual('Silences');
+
+    // Order silences so most recently updated one is on top
+    element(by.cssContainingText('.datatable-header-cell-label.draggable', 'Updated'))
```

Comment:
As discussed before, I think we should filter the table with a specific string to be sure we are testing the correct Silence.
We can't be sure some other test won't change a Silence and that one will be the most recent.